.github/workflows/dev-testing.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       - name: Global Setup
         run: |
           python -m pip install -U pip  # Official recommended way
-          pip install pytest-xdist
           git config --global user.email "CI@example.com"
           git config --global user.name "CI Almighty"
           pip install wheel # Needed for scan image
```

Comment:
Keeping `wheel` in dev but not elsewhere?

Reply:
Not necessary since the latest version of scan image finally decided to dish setup.py:

https://gitlab.com/vidriotech/scanimagetiffreader-python/-/merge_requests/2/diffs

Reply:
So are you intending to remove it from this line or still keeping it?

Reply:
Ah, sorry, I misread, I will remove it here as well : )

Reply:
Done and if test pass this should be ready to go.